docs/UserGuide.md

```diff
@@ -458,13 +477,19 @@ Example: <br>
 
 * Name is a compulsory field that is case-insensitive but space-sensitive.<br>
 
-* Name must be present in Pooch Planner.<br>
+* Name must be present in PoochPlanner.<br>
 
-* Using unpin command on a contact that has been unpinned do not make any changes to Pooch Planner.<br>
+* Using unpin command on a contact that has been unpinned do not make any changes to PoochPlanner.<br>
```

Comment:
Same grammatical errors here:
1. missing article "the", change to "Using _the_ unpin command..."
2. "~~do~~ _does_ not make any changes..."

Reply:
changed